ERROR - 2021-08-30 05:52:54 --> Duplicate entry 'alejado_fve' for key 'email_UNIQUE'
CRITICAL - 2021-08-30 05:52:54 --> Duplicate entry 'alejado_fve' for key 'email_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2021-08-30 06:02:22 --> Call to a member function insert() on null
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(1541): App\Models\UserModel->storeUserInfo(Array)
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(785): CodeIgniter\BaseModel->trigger('afterInsert', Array)
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#3 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2021-08-30 06:09:58 --> Duplicate entry 'alejado_fve' for key 'email_UNIQUE'
CRITICAL - 2021-08-30 06:09:58 --> Duplicate entry 'alejado_fve' for key 'email_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2021-08-30 06:10:35 --> Call to a member function insert() on null
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(1541): App\Models\UserModel->storeUserInfo(Array)
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(785): CodeIgniter\BaseModel->trigger('afterInsert', Array)
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#3 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2021-08-30 06:10:50 --> Duplicate entry 'alejado_fve' for key 'email_UNIQUE'
CRITICAL - 2021-08-30 06:10:50 --> Duplicate entry 'alejado_fve' for key 'email_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2021-08-30 06:11:09 --> Call to a member function insert() on null
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(1541): App\Models\UserModel->storeUserInfo(Array)
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(785): CodeIgniter\BaseModel->trigger('afterInsert', Array)
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#3 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-08-30 06:19:43 --> Call to a member function insert() on null
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(1541): App\Models\UserModel->storeUserInfo(Array)
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(785): CodeIgniter\BaseModel->trigger('afterInsert', Array)
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#3 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2021-08-30 06:19:44 --> Duplicate entry 'AlejandroRomo3' for key 'username_UNIQUE'
CRITICAL - 2021-08-30 06:19:44 --> Duplicate entry 'AlejandroRomo3' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 06:19:47 --> Duplicate entry 'alejado_fvega@gmail.com' for key 'email_UNIQUE'
CRITICAL - 2021-08-30 06:19:47 --> Duplicate entry 'alejado_fvega@gmail.com' for key 'email_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 06:19:48 --> Duplicate entry 'alejado_fvega@gmail.com' for key 'email_UNIQUE'
CRITICAL - 2021-08-30 06:19:48 --> Duplicate entry 'alejado_fvega@gmail.com' for key 'email_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 06:19:49 --> Duplicate entry 'alejado_fvega@gmail.com' for key 'email_UNIQUE'
CRITICAL - 2021-08-30 06:19:49 --> Duplicate entry 'alejado_fvega@gmail.com' for key 'email_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 06:19:50 --> Duplicate entry 'alejado_fvega@gmail.com' for key 'email_UNIQUE'
CRITICAL - 2021-08-30 06:19:50 --> Duplicate entry 'alejado_fvega@gmail.com' for key 'email_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 06:19:51 --> Duplicate entry 'AlejandroRomo3' for key 'username_UNIQUE'
CRITICAL - 2021-08-30 06:19:51 --> Duplicate entry 'AlejandroRomo3' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2021-08-30 06:20:08 --> Call to a member function insert() on null
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(1541): App\Models\UserModel->storeUserInfo(Array)
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(785): CodeIgniter\BaseModel->trigger('afterInsert', Array)
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#3 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2021-08-30 06:29:43 --> Duplicate entry 'AlejandroRomo2' for key 'username_UNIQUE'
CRITICAL - 2021-08-30 06:29:43 --> Duplicate entry 'AlejandroRomo2' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 06:29:50 --> Duplicate entry 'AlejandroRomo3' for key 'username_UNIQUE'
CRITICAL - 2021-08-30 06:29:50 --> Duplicate entry 'AlejandroRomo3' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2021-08-30 06:30:00 --> Cannot declare class App\Models\UserModelInfo, because the name is already in use
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
ERROR - 2021-08-30 06:30:45 --> Duplicate entry 'alejado_fvega@gmail.' for key 'email_UNIQUE'
CRITICAL - 2021-08-30 06:30:45 --> Duplicate entry 'alejado_fvega@gmail.' for key 'email_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 06:30:50 --> Duplicate entry 'AlejandroRomo2' for key 'username_UNIQUE'
CRITICAL - 2021-08-30 06:30:50 --> Duplicate entry 'AlejandroRomo2' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 06:30:52 --> Duplicate entry 'AlejandroRomo3' for key 'username_UNIQUE'
CRITICAL - 2021-08-30 06:30:52 --> Duplicate entry 'AlejandroRomo3' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 06:30:53 --> Duplicate entry 'AlejandroRomo3' for key 'username_UNIQUE'
CRITICAL - 2021-08-30 06:30:53 --> Duplicate entry 'AlejandroRomo3' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 06:30:53 --> Duplicate entry 'AlejandroRomo2' for key 'username_UNIQUE'
CRITICAL - 2021-08-30 06:30:53 --> Duplicate entry 'AlejandroRomo2' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 06:30:54 --> Duplicate entry 'alejado_fvega@gmail.' for key 'email_UNIQUE'
CRITICAL - 2021-08-30 06:30:54 --> Duplicate entry 'alejado_fvega@gmail.' for key 'email_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 06:30:54 --> Duplicate entry 'alejado_fvega@gmail.' for key 'email_UNIQUE'
CRITICAL - 2021-08-30 06:30:54 --> Duplicate entry 'alejado_fvega@gmail.' for key 'email_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 06:30:55 --> Duplicate entry 'alejado_fvega@gmail.' for key 'email_UNIQUE'
CRITICAL - 2021-08-30 06:30:55 --> Duplicate entry 'alejado_fvega@gmail.' for key 'email_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 06:30:55 --> Duplicate entry 'AlejandroRomo2' for key 'username_UNIQUE'
CRITICAL - 2021-08-30 06:30:55 --> Duplicate entry 'AlejandroRomo2' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 06:31:04 --> Duplicate entry 'AlejandroRomo3' for key 'username_UNIQUE'
CRITICAL - 2021-08-30 06:31:04 --> Duplicate entry 'AlejandroRomo3' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 06:31:05 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users_info`, CONSTRAINT `fk_users_info_users1` FOREIGN KEY (`users_id`) REFERENCES `users` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION)
CRITICAL - 2021-08-30 06:31:05 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users_info`, CONSTRAINT `fk_users_info_users1` FOREIGN KEY (`users_id`) REFERENCES `users` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION)
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Models\UserModel.php(39): CodeIgniter\Model->insert(Object(App\Entities\UserInfo))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(1541): App\Models\UserModel->storeUserInfo(Array)
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(785): CodeIgniter\BaseModel->trigger('afterInsert', Array)
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#11 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#12 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#13 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#14 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#15 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#16 {main}
ERROR - 2021-08-30 06:36:36 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users_info`, CONSTRAINT `fk_users_info_users1` FOREIGN KEY (`users_id`) REFERENCES `users` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION)
CRITICAL - 2021-08-30 06:36:36 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users_info`, CONSTRAINT `fk_users_info_users1` FOREIGN KEY (`users_id`) REFERENCES `users` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION)
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Models\UserModel.php(39): CodeIgniter\Model->insert(Object(App\Entities\UserInfo))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(1541): App\Models\UserModel->storeUserInfo(Array)
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(785): CodeIgniter\BaseModel->trigger('afterInsert', Array)
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#11 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#12 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#13 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#14 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#15 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#16 {main}
ERROR - 2021-08-30 06:37:06 --> Duplicate entry 'alejado_fvega@gma' for key 'email_UNIQUE'
CRITICAL - 2021-08-30 06:37:06 --> Duplicate entry 'alejado_fvega@gma' for key 'email_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 06:37:17 --> Duplicate entry 'AlejandroRomo2' for key 'username_UNIQUE'
CRITICAL - 2021-08-30 06:37:17 --> Duplicate entry 'AlejandroRomo2' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 06:37:19 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users_info`, CONSTRAINT `fk_users_info_users1` FOREIGN KEY (`users_id`) REFERENCES `users` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION)
CRITICAL - 2021-08-30 06:37:19 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users_info`, CONSTRAINT `fk_users_info_users1` FOREIGN KEY (`users_id`) REFERENCES `users` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION)
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Models\UserModel.php(39): CodeIgniter\Model->insert(Object(App\Entities\UserInfo))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(1541): App\Models\UserModel->storeUserInfo(Array)
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(785): CodeIgniter\BaseModel->trigger('afterInsert', Array)
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#11 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#12 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#13 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#14 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#15 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#16 {main}
ERROR - 2021-08-30 06:38:03 --> Duplicate entry 'alejado_fvega@gm' for key 'email_UNIQUE'
CRITICAL - 2021-08-30 06:38:03 --> Duplicate entry 'alejado_fvega@gm' for key 'email_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2021-08-30 06:38:23 --> `App\Models\UsersModelInfo` model class does not specify a Primary Key.
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(591): CodeIgniter\Exceptions\ModelException::forNoPrimaryKey('App\\Models\\User...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(251): CodeIgniter\Model->builder()
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#4 C:\laragon\www\v1\app\Models\UserModel.php(39): CodeIgniter\Model->insert(Object(App\Entities\UserInfo))
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(1541): App\Models\UserModel->storeUserInfo(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(785): CodeIgniter\BaseModel->trigger('afterInsert', Array)
#7 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#8 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#11 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2021-08-30 06:38:41 --> Duplicate entry 'alejado_fvega@gm' for key 'email_UNIQUE'
CRITICAL - 2021-08-30 06:38:41 --> Duplicate entry 'alejado_fvega@gm' for key 'email_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 06:38:54 --> Duplicate entry 'alejado_fvega@gm' for key 'email_UNIQUE'
CRITICAL - 2021-08-30 06:38:54 --> Duplicate entry 'alejado_fvega@gm' for key 'email_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 06:39:06 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users_info`, CONSTRAINT `fk_users_info_users1` FOREIGN KEY (`users_id`) REFERENCES `users` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION)
CRITICAL - 2021-08-30 06:39:06 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users_info`, CONSTRAINT `fk_users_info_users1` FOREIGN KEY (`users_id`) REFERENCES `users` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION)
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Models\UserModel.php(39): CodeIgniter\Model->insert(Object(App\Entities\UserInfo))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(1541): App\Models\UserModel->storeUserInfo(Array)
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(785): CodeIgniter\BaseModel->trigger('afterInsert', Array)
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#11 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#12 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#13 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#14 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#15 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#16 {main}
ERROR - 2021-08-30 06:42:10 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users_info`, CONSTRAINT `fk_users_info_users1` FOREIGN KEY (`users_id`) REFERENCES `users` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION)
CRITICAL - 2021-08-30 06:42:10 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users_info`, CONSTRAINT `fk_users_info_users1` FOREIGN KEY (`users_id`) REFERENCES `users` (`id`) ON DELETE NO ACTION ON UPDATE NO ACTION)
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Models\UserModel.php(39): CodeIgniter\Model->insert(Object(App\Entities\UserInfo))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(1541): App\Models\UserModel->storeUserInfo(Array)
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(785): CodeIgniter\BaseModel->trigger('afterInsert', Array)
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#11 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#12 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#13 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#14 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#15 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#16 {main}
ERROR - 2021-08-30 06:43:08 --> Duplicate entry 'AlejandroRomo3' for key 'username_UNIQUE'
CRITICAL - 2021-08-30 06:43:08 --> Duplicate entry 'AlejandroRomo3' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 06:43:15 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users_info`, CONSTRAINT `fk_users_info_users1` FOREIGN KEY (`users_id`) REFERENCES `users` (`id`) ON DELETE CASCADE ON UPDATE CASCADE)
CRITICAL - 2021-08-30 06:43:15 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users_info`, CONSTRAINT `fk_users_info_users1` FOREIGN KEY (`users_id`) REFERENCES `users` (`id`) ON DELETE CASCADE ON UPDATE CASCADE)
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Models\UserModel.php(39): CodeIgniter\Model->insert(Object(App\Entities\UserInfo))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(1541): App\Models\UserModel->storeUserInfo(Array)
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(785): CodeIgniter\BaseModel->trigger('afterInsert', Array)
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#11 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#12 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#13 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#14 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#15 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#16 {main}
ERROR - 2021-08-30 06:48:34 --> Duplicate entry 'AlejandroRomo2' for key 'username_UNIQUE'
CRITICAL - 2021-08-30 06:48:34 --> Duplicate entry 'AlejandroRomo2' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 06:48:41 --> Duplicate entry 'AlejandroRomo4' for key 'username_UNIQUE'
CRITICAL - 2021-08-30 06:48:41 --> Duplicate entry 'AlejandroRomo4' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 06:48:44 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users_info`, CONSTRAINT `fk_users_info_users1` FOREIGN KEY (`users_id`) REFERENCES `users` (`id`) ON DELETE CASCADE ON UPDATE CASCADE)
CRITICAL - 2021-08-30 06:48:44 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users_info`, CONSTRAINT `fk_users_info_users1` FOREIGN KEY (`users_id`) REFERENCES `users` (`id`) ON DELETE CASCADE ON UPDATE CASCADE)
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Models\UserModel.php(39): CodeIgniter\Model->insert(Object(App\Entities\UserInfo))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(1541): App\Models\UserModel->storeUserInfo(Array)
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(785): CodeIgniter\BaseModel->trigger('afterInsert', Array)
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#11 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#12 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#13 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#14 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#15 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#16 {main}
ERROR - 2021-08-30 06:52:37 --> Duplicate entry 'AlejandroRomo2' for key 'username_UNIQUE'
CRITICAL - 2021-08-30 06:52:37 --> Duplicate entry 'AlejandroRomo2' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 06:52:39 --> Duplicate entry 'AlejandroRomo4' for key 'username_UNIQUE'
CRITICAL - 2021-08-30 06:52:39 --> Duplicate entry 'AlejandroRomo4' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 06:52:39 --> Duplicate entry 'AlejandroRomo1' for key 'username_UNIQUE'
CRITICAL - 2021-08-30 06:52:39 --> Duplicate entry 'AlejandroRomo1' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 06:52:40 --> Duplicate entry 'AlejandroRomo5' for key 'username_UNIQUE'
CRITICAL - 2021-08-30 06:52:40 --> Duplicate entry 'AlejandroRomo5' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 06:52:41 --> Duplicate entry 'AlejandroRomo5' for key 'username_UNIQUE'
CRITICAL - 2021-08-30 06:52:41 --> Duplicate entry 'AlejandroRomo5' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 06:52:42 --> Duplicate entry 'AlejandroRomo1' for key 'username_UNIQUE'
CRITICAL - 2021-08-30 06:52:42 --> Duplicate entry 'AlejandroRomo1' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 06:52:45 --> Duplicate entry 'AlejandroRomo3' for key 'username_UNIQUE'
CRITICAL - 2021-08-30 06:52:45 --> Duplicate entry 'AlejandroRomo3' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 06:52:52 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users_info`, CONSTRAINT `fk_users_info_users1` FOREIGN KEY (`users_id`) REFERENCES `users` (`id`) ON DELETE CASCADE ON UPDATE CASCADE)
CRITICAL - 2021-08-30 06:52:52 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users_info`, CONSTRAINT `fk_users_info_users1` FOREIGN KEY (`users_id`) REFERENCES `users` (`id`) ON DELETE CASCADE ON UPDATE CASCADE)
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Models\UserModel.php(39): CodeIgniter\Model->insert(Object(App\Entities\UserInfo))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(1541): App\Models\UserModel->storeUserInfo(Array)
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(785): CodeIgniter\BaseModel->trigger('afterInsert', Array)
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#11 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#12 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#13 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#14 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#15 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#16 {main}
ERROR - 2021-08-30 07:04:47 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users_info`, CONSTRAINT `fk_users_info_users1` FOREIGN KEY (`users_id`) REFERENCES `users` (`id`) ON DELETE CASCADE ON UPDATE CASCADE)
CRITICAL - 2021-08-30 07:04:47 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users_info`, CONSTRAINT `fk_users_info_users1` FOREIGN KEY (`users_id`) REFERENCES `users` (`id`) ON DELETE CASCADE ON UPDATE CASCADE)
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Models\UserModel.php(39): CodeIgniter\Model->insert(Object(App\Entities\UserInfo))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(1541): App\Models\UserModel->storeUserInfo(Array)
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(785): CodeIgniter\BaseModel->trigger('afterInsert', Array)
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#11 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#12 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#13 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#14 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#15 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#16 {main}
ERROR - 2021-08-30 07:07:33 --> Duplicate entry 'AleandroRomo1' for key 'username_UNIQUE'
CRITICAL - 2021-08-30 07:07:33 --> Duplicate entry 'AleandroRomo1' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 07:07:45 --> Duplicate entry 'AleandroRomo1' for key 'username_UNIQUE'
CRITICAL - 2021-08-30 07:07:45 --> Duplicate entry 'AleandroRomo1' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 07:07:47 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users_info`, CONSTRAINT `FK_users_info_users` FOREIGN KEY (`users_id`) REFERENCES `users` (`id`) ON DELETE CASCADE ON UPDATE CASCADE)
CRITICAL - 2021-08-30 07:07:47 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users_info`, CONSTRAINT `FK_users_info_users` FOREIGN KEY (`users_id`) REFERENCES `users` (`id`) ON DELETE CASCADE ON UPDATE CASCADE)
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Models\UserModel.php(39): CodeIgniter\Model->insert(Object(App\Entities\UserInfo))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(1541): App\Models\UserModel->storeUserInfo(Array)
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(785): CodeIgniter\BaseModel->trigger('afterInsert', Array)
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#11 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#12 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#13 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#14 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#15 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#16 {main}
ERROR - 2021-08-30 07:13:49 --> Duplicate entry 'AleandroRomo3' for key 'username_UNIQUE'
CRITICAL - 2021-08-30 07:13:49 --> Duplicate entry 'AleandroRomo3' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 07:13:51 --> Duplicate entry 'alejad_fv' for key 'email_UNIQUE'
CRITICAL - 2021-08-30 07:13:51 --> Duplicate entry 'alejad_fv' for key 'email_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 07:13:52 --> Duplicate entry 'alejad_fv' for key 'email_UNIQUE'
CRITICAL - 2021-08-30 07:13:52 --> Duplicate entry 'alejad_fv' for key 'email_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 07:14:14 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users_info`, CONSTRAINT `FK_users_info_users` FOREIGN KEY (`users_id`) REFERENCES `users` (`id`) ON DELETE CASCADE ON UPDATE CASCADE)
CRITICAL - 2021-08-30 07:14:14 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users_info`, CONSTRAINT `FK_users_info_users` FOREIGN KEY (`users_id`) REFERENCES `users` (`id`) ON DELETE CASCADE ON UPDATE CASCADE)
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Models\UserModel.php(39): CodeIgniter\Model->insert(Object(App\Entities\UserInfo))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(1541): App\Models\UserModel->storeUserInfo(Array)
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(785): CodeIgniter\BaseModel->trigger('afterInsert', Array)
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#11 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#12 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#13 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#14 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#15 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#16 {main}
ERROR - 2021-08-30 07:16:43 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users_info`, CONSTRAINT `FK_users_info_users` FOREIGN KEY (`users_id`) REFERENCES `users` (`id`) ON DELETE CASCADE ON UPDATE CASCADE)
CRITICAL - 2021-08-30 07:16:43 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users_info`, CONSTRAINT `FK_users_info_users` FOREIGN KEY (`users_id`) REFERENCES `users` (`id`) ON DELETE CASCADE ON UPDATE CASCADE)
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Models\UserModel.php(39): CodeIgniter\Model->insert(Object(App\Entities\UserInfo))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(1541): App\Models\UserModel->storeUserInfo(Array)
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(785): CodeIgniter\BaseModel->trigger('afterInsert', Array)
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#11 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#12 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#13 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#14 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#15 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#16 {main}
ERROR - 2021-08-30 08:37:54 --> Duplicate entry 'alejad_fv' for key 'email_UNIQUE'
CRITICAL - 2021-08-30 08:37:54 --> Duplicate entry 'alejad_fv' for key 'email_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 08:38:16 --> Duplicate entry 'AleandroRomo1' for key 'username_UNIQUE'
CRITICAL - 2021-08-30 08:38:16 --> Duplicate entry 'AleandroRomo1' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 08:38:19 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users_info`, CONSTRAINT `FK_users_info_users` FOREIGN KEY (`users_id`) REFERENCES `users` (`id`) ON DELETE CASCADE ON UPDATE CASCADE)
CRITICAL - 2021-08-30 08:38:19 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users_info`, CONSTRAINT `FK_users_info_users` FOREIGN KEY (`users_id`) REFERENCES `users` (`id`) ON DELETE CASCADE ON UPDATE CASCADE)
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Models\UserModel.php(39): CodeIgniter\Model->insert(Object(App\Entities\UserInfo))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(1541): App\Models\UserModel->storeUserInfo(Array)
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(785): CodeIgniter\BaseModel->trigger('afterInsert', Array)
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#11 C:\laragon\www\v1\app\Controllers\Auth\login.php(39): CodeIgniter\Model->insert(Object(App\Entities\User))
#12 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#13 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#14 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#15 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#16 {main}
ERROR - 2021-08-30 08:39:56 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users_info`, CONSTRAINT `FK_users_info_users` FOREIGN KEY (`users_id`) REFERENCES `users` (`id`) ON DELETE CASCADE ON UPDATE CASCADE)
CRITICAL - 2021-08-30 08:39:56 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users_info`, CONSTRAINT `FK_users_info_users` FOREIGN KEY (`users_id`) REFERENCES `users` (`id`) ON DELETE CASCADE ON UPDATE CASCADE)
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Models\UserModel.php(39): CodeIgniter\Model->insert(Object(App\Entities\UserInfo))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(1541): App\Models\UserModel->storeUserInfo(Array)
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(785): CodeIgniter\BaseModel->trigger('afterInsert', Array)
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#11 C:\laragon\www\v1\app\Controllers\Auth\login.php(40): CodeIgniter\Model->insert(Object(App\Entities\User))
#12 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#13 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#14 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#15 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#16 {main}
ERROR - 2021-08-30 08:39:59 --> Duplicate entry 'alejda_fv' for key 'email_UNIQUE'
CRITICAL - 2021-08-30 08:39:59 --> Duplicate entry 'alejda_fv' for key 'email_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(40): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 08:42:08 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users_info`, CONSTRAINT `FK_users_info_users` FOREIGN KEY (`users_id`) REFERENCES `users` (`id`) ON DELETE CASCADE ON UPDATE CASCADE)
CRITICAL - 2021-08-30 08:42:08 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users_info`, CONSTRAINT `FK_users_info_users` FOREIGN KEY (`users_id`) REFERENCES `users` (`id`) ON DELETE CASCADE ON UPDATE CASCADE)
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Models\UserModel.php(39): CodeIgniter\Model->insert(Object(App\Entities\UserInfo))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(1541): App\Models\UserModel->storeUserInfo(Array)
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(785): CodeIgniter\BaseModel->trigger('afterInsert', Array)
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#11 C:\laragon\www\v1\app\Controllers\Auth\login.php(40): CodeIgniter\Model->insert(Object(App\Entities\User))
#12 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#13 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#14 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#15 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#16 {main}
CRITICAL - 2021-08-30 08:44:06 --> Creating default object from empty value
#0 C:\laragon\www\v1\app\Models\UserModel.php(37): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Creating defaul...', 'C:\\laragon\\www\\...', 37, Array)
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(1541): App\Models\UserModel->storeUserInfo(Array)
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(785): CodeIgniter\BaseModel->trigger('afterInsert', Array)
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#4 C:\laragon\www\v1\app\Controllers\Auth\login.php(35): CodeIgniter\Model->insert(Object(App\Entities\User))
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#7 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2021-08-30 08:47:27 --> Duplicate entry 'AleandroRomo1' for key 'username_UNIQUE'
CRITICAL - 2021-08-30 08:47:27 --> Duplicate entry 'AleandroRomo1' for key 'username_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Controllers\Auth\login.php(35): CodeIgniter\Model->insert(Object(App\Entities\User))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-08-30 08:50:05 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users_info`, CONSTRAINT `FK_users_info_users` FOREIGN KEY (`users_id`) REFERENCES `users` (`id`) ON DELETE CASCADE ON UPDATE CASCADE)
CRITICAL - 2021-08-30 08:50:05 --> Cannot add or update a child row: a foreign key constraint fails (`usuarios`.`users_info`, CONSTRAINT `FK_users_info_users` FOREIGN KEY (`users_id`) REFERENCES `users` (`id`) ON DELETE CASCADE ON UPDATE CASCADE)
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\v1\app\Models\UserModel.php(39): CodeIgniter\Model->insert(Object(App\Entities\UserInfo))
#8 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(1541): App\Models\UserModel->storeUserInfo(Array)
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(785): CodeIgniter\BaseModel->trigger('afterInsert', Array)
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#11 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(687): CodeIgniter\Model->insert(Object(App\Entities\User), false)
#12 C:\laragon\www\v1\app\Controllers\Auth\login.php(40): CodeIgniter\BaseModel->save(Object(App\Entities\User))
#13 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#14 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#15 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#16 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#17 {main}
ERROR - 2021-08-30 08:58:19 --> Duplicate entry 'alkmkejka_fv' for key 'email_UNIQUE'
CRITICAL - 2021-08-30 08:58:19 --> Duplicate entry 'alkmkejka_fv' for key 'email_UNIQUE'
#0 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(329): mysqli->query('INSERT INTO `us...')
#1 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseConnection.php(647): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2277): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(259): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(774): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\v1\vendor\codeigniter4\framework\system\Model.php(686): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\laragon\www\v1\vendor\codeigniter4\framework\system\BaseModel.php(687): CodeIgniter\Model->insert(Object(App\Entities\User), false)
#8 C:\laragon\www\v1\app\Controllers\Auth\login.php(40): CodeIgniter\BaseModel->save(Object(App\Entities\User))
#9 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(928): App\Controllers\Auth\login->index()
#10 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth\login))
#11 C:\laragon\www\v1\vendor\codeigniter4\framework\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\v1\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
